iroha_http_server/src/lib.rs

```diff
@@ -606,6 +624,49 @@ pub mod http {
             },
         )
     }
+
+    /// Json wrapper which converts http request body to inner type
+    #[derive(Debug, Clone, Eq, PartialEq, Copy)]
+    pub struct Json<T>(pub T);
```

Comment:
iroha_http_server is a general purpose http server library it should not assume that we receive content in JSON. Moreover we usually receive content in parity scale codec... Or am I missing the meaning of this struct?

Reply:
It doesn't assume anything. This type is just a hint for user and us on how to deserialize those type of requests. It is simmilar to what actix_web does

https://docs.rs/actix-web/3.3.2/actix_web/web/struct.Json.html

Reply:
Ah, I see, ok looks good